crates/wasmtime/src/config.rs

```diff
@@ -127,6 +129,14 @@ pub struct Config {
     profiling_strategy: ProfilingStrategy,
     tunables: ConfigTunables,
 
+    /// Runtime configuration for the stack switching feature.
+    /// The structure is defined in the
+    /// `wasmtime_environ::stack_switching` module, so that we can
+    /// hand out the configuration object in the interface of
+    /// `wasmtime_runtime::Store` trait, where the full `Config` type
+    /// is not in scope.
+    pub stack_switching_config: StackSwitchingConfig,
```

Comment:
This'll definitely want to avoid `pub`, but `pub(crate)` is fine to access it throughout this crate.